maintainers/maintainer-list.nix

```diff
@@ -601,6 +601,12 @@
     githubId = 1229027;
     name = "Andreas Schrägle";
   };
+  ajwhouse = {
+    email = "hi@adamhouse.net";
+    github = "ajwhouse";
+    githubId = 56616368;
+    name = "Adam House";
+  };
```

Comment:
Adding yourself to maintainers needs to be a separate commit; `maintainers: add ajwhouse`